pysrc/scripts/sampling.py

```diff
@@ -98,9 +99,9 @@ def clean_results(results_folder: str) -> None:
     for tmp_file in result_files:
         os.remove(os.path.join(os.getcwd(), results_folder, tmp_file))
 
-def get_file_combinations(samples: list[list[tuple[str, str, float]]], config: GlobalConfiguration) \
-        -> list[list[tuple[str, str, float]]]:
-    data_type_dict = {}
+def get_file_combinations(samples: list[tuple[str, str, float]], config: GlobalConfiguration) \
```

Comment:
Are you sure the change to the type of `samples` is incorrect? With this, I cannot fix the types at the bottom of the method.